tensorflow/tensorflow.bzl

```diff
@@ -120,6 +120,11 @@ def if_x86(a):
       "//conditions:default": [],
   })
 
+def if_darwin(a):
+  return select({
+      "//tensorflow:darwin": a,
```

Comment:
Can you please pass the dependency through clean_dep() as it is done in other similar methods.